doc/application/index.rst

```diff
@@ -504,8 +503,9 @@ again.
 
 .. note:: The ``run`` target will use the QEMU binary available from the Zephyr
```

Comment:
This is only true if building with `ZEPHYR_TOOLCHAIN_VARIANT=zephyr` AFAIK. On macOS for example, you can `ninja run` and it will take brew's QEMU.

Reply:
Isn't that implied by the unavailability of the SDK on macOS?

Reply:
Yes. If you set `ZEPHYR_SDK_INSTALL_DIR`, the build system tries to use the host tools from the SDK regardless of if the toolchain is obtained from the SDK. See the Linux page in the getting started guide.

Reply:
OK but is QEMU considered a "host tool" from that ZEPHYR_SDK_INSTALL_DIR perspective? Not according to @carlescufi initial comment above.

Reply:
@marc-hb it is, in the sense that if `ZEPHYR_SDK_INSTALL_DIR` is set, then the `qemu` binary will be executed from there, but if not it defaults to whatever `qemu` is in the `PATH`. This is the same for `dtc`.
@mbolivar I would still add a "if the Zephyr SDK is installed"

Reply:
> @mbolivar I would still add a "if the Zephyr SDK is installed"

I think this whole debate is a bit outside this PR's scope, but I'll add a patch on top to make this clearer while we are editing this paragraph.

Reply:
I think there are nuances still not captured by this note, however there's only so much you can put in a note like this and the addition of the hyperlink to the SDK page clarifies what "installed" means if desired which seems like the best solution. The only statement I would like added (on either page) is the confirmation that QEMU is indeed a "host tool" but no big deal.